### PR TITLE
Update pillow to resolve vulnerability QA-2089

### PR DESCRIPTION
**Pull request by bpierce01**
## Description
- Fixed pillow vulnerability in holmes-py

### CODE REVIEW

File: holmes-py/requirements.txt [hunk 1]
```diff
@@ -30,8 +30,8 @@ mouseinfo==0.1.3
     # via pyautogui
 nodeenv==1.7.0
     # via pre-commit
-pillow==9.4.0
-    # via pyscreeze
+pillow==10.0.1
```
Comment (uc-pperumala) [inline holmes-py/requirements.txt:33]: The dependencies in this file should be auto-generated by using this command:
```bash
pip-compile requirements.in```

Comment (bpierce01) [reply holmes-py/requirements.txt:33]: Yes. I updated requirements.in to tie pillow to that version. Then, I ran "pip-compile requirements.in" to generate the requirements.txt file. This was the result

Comment (uc-pperumala) [reply holmes-py/requirements.txt:33]: That's interesting. I was expecting bunch of auto-generated changes. Thank you!

**Review by uc-pperumala — APPROVED** [review]
Changes look good!

**Pull request merged**